### ISSUE TITLE
Receive audio data at the end of recognition

### ISSUE THREAD

**Issue by Nixellion** [TalAter/annyang#231]
I'd like to be able to operate with recorded audio data after the recognition. 
## Example use:

Get a command from the user. Send received text-command and audio to the server. Let server process audio, for example with Microsoft Cognitive Services, or internally, to get more information about the command: Speaker ID (Who is speaking? For authorization), emotion, etc. And then decide if the user has access to this command or not, based on his voice id.

Or maybe there is some work around for this, like with the interim-results Issue here?

Thanks.

**Comment by JVRibeiro** [comment]
I'm interested on it.

**Comment by BetaStacks** [comment]
I'm interested in this too.

**Comment by Nixellion** [comment]
I ended up using a hack. I was able to achieve it by using recorder.js (one of them anyway) and recording audio on annyang's 'audio start' event and saving it on 'audio end'. If the command was recognized, I then take this saved blob audio, and just put it's blob variable into json, using socketio to send it to the server, but I think that AJAX or something else may also work, depending on what you want to do with the audio.

Variable names may not be correct, just describing the logic here.
